app/controllers/concerns/foreman/controller/smart_proxy_auth.rb

```diff
@@ -4,10 +4,13 @@
 module Foreman::Controller::SmartProxyAuth
   extend ActiveSupport::Concern
 
+  def require_ssl?
+    false
+  end
+
   module ClassMethods
     def add_smart_proxy_filters(actions, options = {})
       skip_before_filter :require_login, :only => actions
-      skip_before_filter :require_ssl, :only => actions
```

Comment:
the `require_ssl?` method should only skip the SSL requirement on `actions`, not controller-wide